### PR TITLE
Style enhancements to improve layout and remove sick/work related tra…

### PR DESCRIPTION
**Pull request by mcromie92**
…vel from booking dropdown

### CODE REVIEW

File: web-app/src/hoc/Layout/nav-menu.js [hunk 1]
```diff
@@ -28,6 +28,7 @@ const NavMenu = ({
       type: 'question',
       showCancelButton: true,
       confirmButtonText: 'Yes, log out',
+      confirmButtonColor: '#0eb5d1',
```
Comment (ghost) [inline web-app/src/hoc/Layout/nav-menu.js]: @William-Unosquare  @paul-matchett this is the only thing I wasn't sure on when looking at it with Michaela, obviously the uno hex variable is being used in styled components but this outside of that scope. Not sure how you guys would recommend going forward here. Obviously I think it's fine like this personally.

Comment (paulmatchett) [reply web-app/src/hoc/Layout/nav-menu.js]: Yes, there is no way of accessing the styled components themed props to inject here. So, yes, this is fine

**Review by paulmatchett — APPROVED** [review]
All looks good to me.

File: web-app/src/hoc/Layout/nav-menu.js (hunk 1 above)
Comment (William-Unosquare) [reply web-app/src/hoc/Layout/nav-menu.js]: @mcromie92 @girwin575 You can access the styled components theme by importing theme object directly from the `styled.js` file in the root directory. Then you could do something like `confirmButtonColor: theme.colors.unoBlue`

**Review by William-Unosquare — CHANGES_REQUESTED** [review]
Great stuff! Just take a quick look at the comment above about importing the theme.

**Comment by mcromie92** [comment]
@William-Unosquare, @girwin575 I have updated the CSS for unoBlue importing the theme

**Review by William-Unosquare — APPROVED** [review]

**Comment by William-Unosquare** [comment]
Looks good!

**Pull request merged**